app/views/calculation/gain/worthWhenSoldForLess.scala.html

```diff
@@ -16,8 +16,12 @@
 
     <h1 class="heading-large">@Messages("calc.resident.shares.worthWhenSoldForLess.question")</h1>
 
+    <p id="information">@Messages("calc.resident.shares.worthWhenSoldForLess.informationText")</p>
+
     @form(action = controllers.routes.GainController.submitWorthWhenSoldForLess) {
 
+        <p class="panel-indent panel-indent--wide panel-indent--gutter">@Messages("calc.resident.shares.worthWhenSoldForLess.jointOwnership")</p>
+
```

Comment:
The message is not displayed on the page. Instead the variable "calc.resident.shares.worthWhenSoldForLess.jointOwnership" is